pkg/util/etcdutil/member.go

```diff
@@ -67,7 +69,11 @@ func (m *Member) ListenPeerURL() string {
 }
 
 func (m *Member) PeerURL() string {
-	return fmt.Sprintf("%s://%s:2380", m.peerScheme(), m.Addr())
+	host := m.Addr()
+	if m.SelfHosted {
+		host = "$(POD_IP)"
```

Comment:
In PeerURLPairs(), it calls this method to construct `initialCluster` string.
For pod "0001", it will set other peers to use its own Pod IP even if they have different Pod IP.

We will need to update Member struct with its pod IP when a new Pod and new member is created.